go/vt/mysqlctl/mycnf.go

```diff
@@ -153,8 +158,20 @@ func normKey(bkey []byte) string {
 
 // ReadMycnf will read an existing my.cnf from disk, and update the passed in Mycnf object
 // with values from the my.cnf on disk.
-func ReadMycnf(mycnf *Mycnf) (*Mycnf, error) {
+func ReadMycnf(mycnf *Mycnf, waitTime time.Duration) (*Mycnf, error) {
 	f, err := os.Open(mycnf.Path)
+	if waitTime != 0 {
+		timeout := time.After(waitTime)
+		for err != nil {
+			select {
+			case <-timeout:
+				return nil, err
+			default:
+				time.Sleep(myCnfWaitRetryTime)
```

Comment:
Use time.NewTicker instead of time.Sleep.

Reply:
Fixed